rpcs3/Emu/Cell/SPUThread.cpp

```diff
@@ -1670,7 +1670,7 @@ void spu_thread::do_mfc(bool wait)
 	u32 fence = 0;
 
 	// Process enqueued commands
-	std::remove_if(mfc_queue + 0, mfc_queue + mfc_size, [&](spu_mfc_cmd& args)
+	static_cast<void>(std::remove_if(mfc_queue + 0, mfc_queue + mfc_size, [&](spu_mfc_cmd& args)
```

Comment:
```suggestion
	+std::remove_if(mfc_queue + 0, mfc_queue + mfc_size, [&](spu_mfc_cmd& args)
```

Reply:
actually.
```suggestion
	0, std::remove_if(mfc_queue + 0, mfc_queue + mfc_size, [&](spu_mfc_cmd& args)
```
better marks the intent.

Reply:
I prefer casting to void, it's more clear what is going on

Reply:
It's also the recommended/standardized way of handling. For example [cppreference](https://en.cppreference.com/w/cpp/language/attributes/nodiscard) has "If, from a discarded-value expression **other than a cast to void,..**"